packages/next/components/postDetail/biasedVotingItem.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import Tooltip from "../tooltip";
+import ValueDisplay from "../valueDisplay";
+
+const Wrapper = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-item: center;
+  width: 100%;
+`;
+
+const LabelWrapper = styled.div`
+  color: #506176;
+  position: relative;
+  width: 50%;
+
+  .label {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+`;
+
+function BiasedVotingItem({ label = "", value, space }) {
+  const displayLabel = <div className="label">{label}</div>;
+
+  return (
+    <Wrapper>
+      <LabelWrapper>
+        {/*
+          TODO: Should refactor, this is not a smart way to show the tooltip or not
+        */}
+        {label.length > 10 ? (
```

Comment:
@YoshiyukiSakura is this necessary? if yes, how many characters should show the tooltip?

Reply:
its not a good approach, since `ABCDE` is much wider than `iiiii` yet they both count as `5`